ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md

```diff
@@ -0,0 +1,241 @@
+[!!Workflows](ActindoWorkFlow)
+
+# Configure the queue types
+
+Configure an arbitrary number of individual queue types to be able to execute certain actions faster. You can configure an unlimited number of queue types with a defined number of workers and the and a defined sequence of how actions are to be executed. The total number of workers that can be assigned to the queue types is limited by the number of booked vCores.
+The *Default* queue type is predefined and is assigned to all actions by default. It can manually be overridden by individual queue types. A different queue type may be assigned for each action.
+
+
+## Create a queue type
+
+Create an individual queue type, assign a certain number of workers to it and define the sequence in which the actions are to be executed.
+
+### Prerequisites
+
+No prerequisites to fulfill.
+
+### Procedure
+
+*Workflows > Queue Types > Tab OVERVIEW*
+
+![Overview](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/Overview.png "[Overview]")
+
+1. Click the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+  The *Create Queue Type* view is displayed.
+
+  ![Create queue type](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/CreateQueueType.png "[Create queue type]")
+
+2. Enter a name for the queue type in the *Name* field.
+
+3. Enter the desired number of workers for the queue type in the *Number of workers* field.
+
+  > [Info] A worker is a job that is executing the action within a process. One worker can only execute one job at the same time. By assigning multiple workers to a queue type, this queue type may execute several jobs at the same time. Consequently, actions with a queue type with more workers are executed faster that actions with queue types with less workers. The total number of workers of all queue types depends on the number of booked vCores.
+
```

Comment:
I find this description a bit contradictory: **"A worker is a job that is executing the action** within a process. **One worker can only execute one job**  job at the same time."

Reply:
That is right...